plugin/.idea/.name

```diff
@@ -0,0 +1 @@
+PlayAssetDelivery
```

Comment:
Can we ignore the files in the `.idea` folder? The godot oculus mobile plugin has it listed in it's [`.gitignore`](https://github.com/GodotVR/godot_oculus_mobile/blob/06f382bcbec0ac4647c7e7cc80724d9d78d092c7/.gitignore#L18).

Reply:
Removed the .idea folder from repository, updated .gitignore file.